content/post/Dabbling_with_react_@_React_India_2019.md

```diff
@@ -0,0 +1,72 @@
+
+GraphQL is much more than efficiently pulling data, and I am glad Nader's talk helped make the case for it as he showcased the powers of AWS Amplify as well as some awesome problems he solved with GraphQL. 
+
+Moving on, I took a break from GraphQL and attended talks by [Ives van Hoorne]() about building [CodeSandbox](https://codesandbox.io
+), and React for teenagers by my roomate, Yash Gupta. Both very inspirational on how to build products for everyone and contributing back to the community. You all should really try CodeSandbox, it's like VsCode but online and more flexible. 
+
+<blockquote class="twitter-tweet" data-conversation="none" data-dnt="true"><p lang="en" dir="ltr">First picture is is off <a href="https://twitter.com/yashguptaz?ref_src=twsrc%5Etfw">@yashguptaz</a> preparing for his session.<br><br>The other one, is the youngest speaker (Just 16) at <a href="https://twitter.com/react_india?ref_src=twsrc%5Etfw">@react_india</a> giving his session to help teenagers join the <a href="https://twitter.com/reactjs?ref_src=twsrc%5Etfw">@reactjs</a> community. <a href="https://t.co/S0Ownxdipi">pic.twitter.com/S0Ownxdipi</a></p>&mdash; Vipul Gupta 🐣 (@vipulgupta2048) <a href="https://twitter.com/vipulgupta2048/status/1177521013799579648?ref_src=twsrc%5Etfw">September 27, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Community is where the heart is
+
+Open-Source was a general theme behind the conference, where people talked from how we can build more inclusive communities to releasing new frameworks for blogging platforms to conflict resolution to this amazing quote by Carolyn in her talk about "Intitutive tooling". 
+
+> "We mistake familiarity for simplicity" - Carolyn Stransky
+
+She argues on the fact that we developers, once we are fluent with a framework/technology, we end up introducing it as "simple". While its simple for you now it might not be to a newcomer due to a steep learning curve or due to improper documentation. ([Picked up from source](https://dev.to/kay_2695px/react-india-2019-1goa))
```

Comment:
```suggestion
She argues on the fact that we developers, once we are fluent with a framework/technology, end up introducing it as "simple". While its simple for you now, it might not be to a newcomer due to a steep learning curve or due to improper documentation. ([Picked up from source](https://dev.to/kay_2695px/react-india-2019-1goa))
```